docs/architecture/adr013-non-interactive-default-rules-for-zero-padding.md

```diff
@@ -0,0 +1,86 @@
+# ADR 013: Non-Interactive Default Rules for Zero Padding
+
+## Changelog
+
+- 03.01.2023: initial draft
+
+## Context
+
+When laying out blobs in the square we create padding to follow the non-interactive default rules. With ADR 009 we decreased padding by aligning the blob in the square to the index of the `MinSquareSize` of the given blob. This is a good improvement but we can do better.
+
+Looking at different ranges of blob sizes we can see that the ratio of blob size to padding is not constant. Insight:  **The ratio of blob size to padding is smaller for smaller blobs and larger for larger blobs.** The bigger the ratio the better.
+
+![Worst Case Padding In Blob Size Range](./assets/worst-case-padding-in-blob-size-range.png)
+
+This means that you get more padding for fever data in small blobs. This is not ideal as we want to have as little padding as possible. As padding is not being paid for there is no incentive to use larger blobs.
```

Comment:
[typo]: "for fever data"

Apologies for no suggestion. I'm not sure what it meant to say.

Reply:
I rephrased